### DISCUSSION TITLE
Release cycle for the plugin

### DISCUSSION THREAD

**Discussion by arpit15** [mitsuba-renderer/mitsuba-blender#75] [General]
Thanks for releasing and maintaining this amazing tool. I am wondering if there is a plan to release versions of this plugin which tracks mitsuba3 versions. Specifically, it might be helpful to create a plan for new releases with major changes like blender mesh refactor and support for rendering.  

I would like to help with constant release and maintainence as this is direclty helpful for my research.

**Comment by bathal1** [comment]
Hi, 
There is currently no plan to have releases that track Mitsuba 3 versions, mostly because most new releases of Mitsuba 3 bring no changes to the exporter, therefore it seems overkill to keep track of every single release.

On the other hand, sometimes breaking changes are introduced, like the recent changes to the mesh data structure in Blender 3.6. My plan for this is to make a release before and after merging the #63 branch into master.

I do agree that having a more principled release strategy for the exporter would be good, but I'm not certain what it should be. If you have suggestions, I'm happy to read them.

**Comment by arpit15** [comment]
I agree with you that there is no need to release after every version of Mitsuba. I have following suggestions

- It might be useful to start a **next release planning post**. Specifically, mention which tasks are important to put in the next release. Also mention which issues might be addressed by each task.
- It might be useful to put label on issues if it is targetted by the next release

I am interested in contributing in the next release and can create or edit your planning post.
